### DISCUSSION TITLE
Frequent apps not updating

### DISCUSSION THREAD

**Discussion by isle9** [Ulauncher/Ulauncher#1272] [Q&A]
Seems like the list of the top 8 of my most frequent app stays the same. I just re-installed Ubuntu a few days ago, so it should start to change by now. Does anyone have any idea what might have gone wrong? Can I delete some cache / database somewhere to restart the process again?

**Comment by igorsantos07** [comment]
I've also noticed this for quite some time now.

~Given the last release was 8 months ago and no one answered this, I guess we can consider ULauncher abandonware by now?~
Edit: it looks like they're just stuck forever with the v6 upgrade.... I hope it gets released at some point and we're back with a stable and maintained app :(

**Comment by friday** [comment]
I just tested it now. Works perfectly on v6 `~/.local/state/ulauncher/app_starts.json` updates every launch and it updates in real time too (if you bump something up the list you don't need to restart).

But it doesn't care about recency. If you had a favorite browser for 5 years, and switch your browser without uninstalling the old one, the old one will stay on top until you have launched the new browser more times than the old one. It always worked like that though.

**Reply by friday** [reply]
Created an issue for this https://github.com/Ulauncher/Ulauncher/issues/1434

**Reply by igorsantos07** [reply]
I would consider this question answered then, as work on v5 got freeze and the issue is fixed in the next major.
Thanks!
